<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainkey931</class>
 <widget class="QDialog" name="mainkey931">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9.3.1 参数设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>退  出</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>终端参数</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>5</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>商户名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>68</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>107</y>
       <width>89</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>91</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>终端号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>131</y>
       <width>57</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>凭证号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>111</y>
       <width>89</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>131</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>批次号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>48</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>商户号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>24</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>151</y>
       <width>89</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>151</y>
       <width>89</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>91</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>最大笔数</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>通信参数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>80</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>通讯方式</string>
     </property>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>76</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3.GPRS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>53</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2.拨号</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>123</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5.TCP/IP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4.CDMA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>80</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1.RS232</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>0</y>
       <width>150</width>
       <height>190</height>
      </rect>
     </property>
     <property name="title">
      <string>通讯参数</string>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>150</width>
        <height>160</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_3">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>68</x>
          <y>20</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>22</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>中心号1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>52</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>中心号2</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>81</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>中心号3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>108</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>管理号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>108</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>134</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分机号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>135</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>40</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>交易IP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>40</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备份IP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>54</x>
          <y>77</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>54</x>
          <y>100</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>50</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>主机端口</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备份端口</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>输入APN </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_16">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>157</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AR PL UKai CN</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>数据加密</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>类型参数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>应用类别</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>20</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PBOC</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>38</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EMV</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>121</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>打印屏蔽设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>内卡</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>外卡</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>内外卡</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>106</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>打印屏蔽卡号</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>40</y>
        <width>119</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>预授权屏蔽卡号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>选择屏蔽卡种</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>终端类型</string>
     </property>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>17</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>百货</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>37</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>宾馆</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>57</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>餐饮</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>手机无卡预约</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>分期付款</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
